package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.7",
+    "axios": "^1.6.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.18.0"
+  },
+  "scripts": {
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --loader:.js=jsx"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.2",
```

Comment:
- [ ] [REQUIRED] Good job so far! Please required config files should be set up as showed in [the totorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a): and this file created at your working place 🙏🏼 

`babel.config.js`

![image](https://github.com/lRebornsl/hello-rails-react/assets/63915024/a3e9c205-ea8e-4214-a598-51c8eafb865e)

Reply:
Thanks for your feedback 👍🏼 

Solved 😄